examples/ensemble/plot_gradient_boosting_regularization.py

```diff
@@ -1,93 +0,0 @@
-"""
```

Comment:
You should not remove this example :)
Can you revert this change.

Reply:
ok

Reply:
I saw but you should not remove this file. Currently, it is tagged as deleted from your side in your `git` history. You need to add the original file back.

Reply:
done